packages/documentation/src/layouts/module-components.js

```diff
@@ -15,13 +15,26 @@ const MyCodeComponent = ({ children }) => (
 );
 
 export default class MDXRuntimeTest extends Component {
+  sortedReqiuredProps() {
```

Comment:
Added this so that we can have the required props on top of the list of props in the documentation.